### ISSUE TITLE
Error running the example

### ISSUE THREAD

**Issue by mwildehahn** [dferber90/webapp#3]
Following the README, I get:

```
$ npm start

> webapp-starter-pack@1.0.0 start /Users/mhahn/open_source/webapp-starter-pack
> npm run clean && node webpack/dev.js


> webapp-starter-pack@1.0.0 clean /Users/mhahn/open_source/webapp-starter-pack
> rimraf build/*

compiled server
server listening
ReferenceError: Promise is not defined
    at exports.default (/Users/mhahn/open_source/webapp-starter-pack/build/webpack:/app/server/generatePage.js:12:14)
    at /Users/mhahn/open_source/webapp-starter-pack/build/webpack:/app/server/server.js:15:3
    at Layer.handle [as handle_request] (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/route.js:131:13)
    at Route.dispatch (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/layer.js:95:5)
    at /Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/index.js:277:22
    at param (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/index.js:349:14)
    at param (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/index.js:365:14)
    at Function.process_params (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/index.js:410:3)
```

**Comment by dferber90** [comment]
I don't know why this occurs for you. `Promise` should be polyfilled by Babel.

Did you run `npm install` before `npm start`?

**Comment by mwildehahn** [comment]
yup:

```
$ npm install
npm WARN package.json webapp-starter-pack@1.0.0 No repository field.
npm WARN package.json babel-core@5.8.21 No README data
npm WARN package.json redux-react-router@0.2.1 No repository field.

$ npm start

> webapp-starter-pack@1.0.0 start /Users/mhahn/open_source/webapp-starter-pack
> npm run clean && node webpack/dev.js


> webapp-starter-pack@1.0.0 clean /Users/mhahn/open_source/webapp-starter-pack
> rimraf build/*

compiled server
server listening
ReferenceError: Promise is not defined
    at exports.default (/Users/mhahn/open_source/webapp-starter-pack/build/webpack:/app/server/generatePage.js:12:14)
    at /Users/mhahn/open_source/webapp-starter-pack/build/webpack:/app/server/server.js:15:3
    at Layer.handle [as handle_request] (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/route.js:131:13)
    at Route.dispatch (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/layer.js:95:5)
    at /Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/index.js:277:22
    at param (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/index.js:349:14)
    at param (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/index.js:365:14)
    at Function.process_params (/Users/mhahn/open_source/webapp-starter-pack/node_modules/express/lib/router/index.js:410:3)
```

**Comment by dferber90** [comment]
I am sorry but I do not know why it is not working for you. This has worked for other people so it must be something "unique" to your setup.

I have two wild guesses

1) Babel does not polyfill Promises in node environments and you are using a node version that does not have them natively. I have tested on v0.12.2 and v0.12.7

```
$ node -v
v0.12.7
```

2) I installed something globally that should have been added as a dependency in `package.json`. Below is a list of my globally installed packages related to this project, but I am not sure this is helping.

```
$ npm ls -g -depth 0
/usr/local/lib
├── babel@5.4.7
├── babel-eslint@3.1.5
├── eslint@0.24.1
├── eslint-plugin-react@2.7.0
├── faucet@0.0.1
├── jsxhint@0.14.0
├── node-inspector@0.11.1
├── node-libs-browser@0.5.2
├── npm@2.7.4
├── react-tools@0.13.3
├── webpack@1.10.0
└── webpack-dev-server@1.10.1
```
